lib/ci-helper.ts

```diff
@@ -554,6 +559,17 @@ export class CIHelper {
                 await addComment(`Submitted as [${
                     coverMid}](https://public-inbox.org/git/${coverMid})`);
             } else if (command === "/allow") {
+                if (!argument) {
```

Comment:
It might make sense, for readability, to use something like `const accountName = argument || await getPRAuthor();`, and leave `argument` as a constant.

Reply:
Okay.